### DISCUSSION TITLE
Issues with applying for GitHub education

### DISCUSSION THREAD

**Discussion by MrEpiko** [community/community#133330] [GitHub Education]
### Hi everyone,

Hello,
I am trying to apply for GitHub education, however, I am facing couple of issues. First of all, back camera on my phone is for some reason not working on GitHub's website and, as I also have translated image of my Student's report card (due to it being on foreign language) on my PC, it is basically impossible to take a shot of both my monitor and my card in real life. What can I do?

**Comment by romhenri** [comment]
Tip:

Try use another browser in phone. Some browsers like Opera allows you choose what "cam" you will use.

**Reply by MrEpiko** [reply]
I tried from Chrome and Safari, none worked. I do get an option to choose camera but nothing other than “Front camera” option is working.
![image](https://github.com/user-attachments/assets/fe6a280d-03b1-4734-9ad1-8b046870cec7)

**Reply by romhenri** [reply]
Try see in your browser the cam options.

**Reply by MrEpiko** [reply]
Nope, I tried and still nothing..

**Reply by romhenri** [reply]
I think maybe a phone problem. I would try another phone of a friend just to do it and after remove your login.

**Reply by MrEpiko** [reply]
I did it from the other phone but it’s still impossible to take a shot of everything with high precision. I tried 3 times and all 3 times my application was denied. It even started telling me to setup my billing and personal information properly but I already have them set up properly. Is there any way for a human being to scan my application as whichever system is used right now, it’s not good and it surely can’t understand student’s report card that’s written in Serbian cyrilic..
